### ISSUE TITLE
Can not exec rails command

### ISSUE THREAD

**Issue by minakawa-daiki** [docker-library/ruby#211]
Version:  Docker Image `ruby:2.4.4`

https://github.com/docker-library/ruby/commit/ee0b15d76c92ad221caf9afaf9b8827e7fe39f9a#diff-ff2de49361eed004fa5d18a474ece86a

Please check this commit diff.

I can't exec `rails s` after `bundle install` recently.

I think it's caused by `$BUNDLE_BIN` removed.

And not exist path "`$BUNDLE_PATH/gems/bin`".

**Comment by jozko** [comment]
Ran into a similar problem using Docker image `ruby:2.5.1-stretch` - at the build time we got this lovely error (nothing else changed code wise, only the upstream base image got updated a few hours ago):

```
^@^@docker: Error response from daemon: oci runtime error: container_linux.go:265: starting container process caused "exec: \"rubocop\": executable file not found in $PATH".
Exited with code 127
```

The root cause was incorrect `PATH` set to `/usr/local/bundle/gems/bin` where it should've been set to `/usr/local/bundle/bin`. After updating the PATH in out Dockerfile everything is green again. Not sure why the `PATH` got updated in the upstream though.

**Comment by fwininger** [comment]
Same problem, with ruby:slim

It's seem to be a problem introduced with : 
https://github.com/docker-library/ruby/commit/77515a062795c609e645bf84d5e09f553ec9deef#diff-74bcd4eb147d76d56d72bb7b43848b20
https://github.com/docker-library/ruby/commit/ee0b15d76c92ad221caf9afaf9b8827e7fe39f9a#diff-74bcd4eb147d76d56d72bb7b43848b20

**Comment by fwininger** [comment]
The completed discussion is here : 

https://github.com/bundler/bundler/pull/6469#issuecomment-383235438

**Comment by fwininger** [comment]
Duplicate with #209

**Comment by minakawa-daiki** [comment]
@fwininger My bad..., Thank to find Duplicate issue!

**Comment by yosifkit** [comment]
Fixed by #209.
